src/canopy-webpack-config.js

```diff
@@ -10,6 +10,12 @@ if (process.argv.some(arg => arg.includes('webpack-dev-server'))) {
   isDevServer = true
 }
 
+const hostIndex = process.argv.findIndex(arg => arg === "--host")
+const host = hostIndex >= 0 && process.argv[hostIndex + 1] ? process.argv[hostIndex + 1] : "0.0.0.0"
+
+const portIndex = process.argv.findIndex(arg => arg === "--port")
+const port = portIndex >= 0 && process.argv[portIndex + 1] ? process.argv[portIndex + 1] : "8080"
```

Comment:
Do you really need to do this? By default doesn't webpack take CLI config precedence over the config file?